### PR TITLE
update prepare_worker for autoprocessing

### PR DESCRIPTION
**Pull request by fabiansinz**

### CODE REVIEW

File: matlab/scripts/segmentation_worker.m [hunk 1]
```diff
@@ -1,5 +1,8 @@
-r  = experiment.Session & 'session_date>"2016-02"';
+r  = experiment.Session & 'session_date>"2016-02"' & experiment.AutoProcessing;
+
 while true
-    parpopulate(preprocess.ExtractRaw, r)
+    while count((preprocess.ExtractRaw().popRel & r) - preprocess.ExtractRaw)
```
Comment (dimitri-yatsenko) [inline matlab/scripts/segmentation_worker.m]: You can replace this line with
```matlab
while progress(preprocess.ExtractRaw)
```

File: matlab/scripts/prepare_worker.m [hunk 2]
```diff
@@ -1,5 +1,11 @@
-r  = experiment.Session & 'session_date>"2016-02"';
+r  = experiment.Session & 'session_date>"2016-02"' & experiment.AutoProcessing;
+
 while true
-    parpopulate(preprocess.Prepare, r)
+    while count((preprocess.Prepare().popRel & r) - preprocess.Prepare)
+    	  parpopulate(preprocess.Prepare, r)
+    end
+    while count((preprocess.Sync().popRel & r) - preprocess.Sync)
```
Comment (dimitri-yatsenko) [inline matlab/scripts/prepare_worker.m]: You can replace this line with
```matlab
while progress(preprocess.Sync)
```

File: matlab/scripts/prepare_worker.m [hunk 3]
```diff
@@ -1,5 +1,11 @@
-r  = experiment.Session & 'session_date>"2016-02"';
+r  = experiment.Session & 'session_date>"2016-02"' & experiment.AutoProcessing;
+
 while true
-    parpopulate(preprocess.Prepare, r)
+    while count((preprocess.Prepare().popRel & r) - preprocess.Prepare)
```
Comment (dimitri-yatsenko) [inline matlab/scripts/prepare_worker.m]: You can replace this line with
```matlab
while progress(preprocess.Prepare)
```

Comment (fabiansinz) [inline matlab/scripts/prepare_worker.m]: I'll add it. I actually checked for that but progress is missing that in it's help string.

**Pull request merged**